### PR TITLE
MWPW-161779: SUSI Link scenario when open in new tab or window

### PR DESCRIPTION
**Pull request by joaquinrivero**
Resolves: [MWPW-161779](https://jira.corp.adobe.com/browse/MWPW-161779)

- Adds default link in to the anchor when user use the right click to open the link in new tab or window avoiding a 404

Test URL:

https://main--dc--adobecom.hlx.page/acrobat/online
https://mwpw-161779--dc--adobecom.hlx.page/acrobat/online

### CODE REVIEW

File: acrobat/scripts/susiAuthHandler.js [hunk 1]
```diff
@@ -1,21 +1,42 @@
...
+export default async function handleImsSusi(susiElems) {
+  const { setLibs } = await import('./utils.js');
+  const miloLibs = setLibs('/libs');
+  const { getConfig } = await import(`${miloLibs}/utils/utils.js`);
+  const config = getConfig();
+
+  // Ensure placeholders are loaded
+  if (!Object.keys(window.mph || {}).length) {
+    const placeholdersPath = `${config.locale.contentRoot}/placeholders.json`;
+    const response = await fetch(placeholdersPath);
```
Comment (TsayAdobe) [inline acrobat/scripts/susiAuthHandler.js:26]: It may go wrong with fetch. I would try catch exceptions.

**Review by TsayAdobe — APPROVED** [review]

File: acrobat/scripts/susiAuthHandler.js [hunk 2]
```diff
@@ -1,21 +1,42 @@
 import susiAnalytics from './alloy/susi-auth-handler.js';
 
-export default function handleImsSusi(susiElems) {
+async function handleButton(link, action) {
```
Comment (TsayAdobe) [inline acrobat/scripts/susiAuthHandler.js:3]: I don't see `await`. Is `async` necessary?

File: acrobat/scripts/susiAuthHandler.js [hunk 3]
```diff
@@ -1,21 +1,42 @@
...
-        window.adobeIMS.signUp();
-      });
+      handleButton(link, 'sign-up');
     } else if (link.href.includes('-sign-in')) {
-      link.addEventListener('click', (e) => {
-        e.preventDefault();
-        susiAnalytics(link.text, 'sign-in', window.browser.isMobile ? 'mobile' : 'desktop');
-        window.adobeIMS.signIn();
-      });
+      handleButton(link, 'sign-in');
```
Comment (TsayAdobe) [inline acrobat/scripts/susiAuthHandler.js:39]: How about
```
const match = link.href.match(/-(sign-up|sign-in)/);
if (match) {
  handleButton(link, match[1]);
}
```

File: acrobat/scripts/susiAuthHandler.js [hunk 4]
```diff
@@ -1,21 +1,42 @@
 import susiAnalytics from './alloy/susi-auth-handler.js';
 
-export default function handleImsSusi(susiElems) {
+async function handleButton(link, action) {
+  link.classList.add('susi-cta');
+  link.href = window.mph['susi-default-redirect'] || 'https://acrobat.adobe.com/link/home/';
+  link.addEventListener('click', (e) => {
+    e.preventDefault();
+    susiAnalytics(link.text, `${action}`, window.browser.isMobile ? 'mobile' : 'desktop');
```
Comment (TsayAdobe) [inline acrobat/scripts/susiAuthHandler.js:8]: Unnecessary string interpolation
```
`${action}`
```

**Review by tedAir — APPROVED** [review]
Verified on Chrome, FF, Safari, PASS.

**Pull request merged**